local.properties

```diff
@@ -0,0 +1,8 @@
+## This file must *NOT* be checked into Version Control Systems,
+# as it contains information specific to your local configuration.
+#
+# Location of the SDK. This is only used by Gradle.
+# For customization when using a Version Control System, please read the
+# header note.
+#Tue Feb 25 15:00:55 CET 2020
+sdk.dir=/Users/marandaneto/Library/Android/sdk
```

Comment:
This file should be ignored?

Reply:
@dzwicker it's outdated, when this repo has been moved, we lost the gitignore, but it already has been fixed.